README.md

```diff
@@ -136,6 +137,22 @@ client.send_event(
     }
   }
 )
+
+# Send a voucher event
+
+client.send_event(
+  name: :voucher,
+  payload: {
+    voucher_code: 'TEST123',
+    expiry: 1497484799,
```

Comment:
This is valid, but we allow for `expiry: Time.now`. `Time` objects get detected and converted to epoch.

https://github.com/deliveroo/ravelin-ruby/blob/master/lib/ravelin/ravelin_object.rb#L48

Same for the `created_at` key/value.